parameters/src/testnet/resources/transfer_private.metadata

```diff
@@ -1,6 +1,6 @@
 {
   "prover_checksum": "6733de36be3291a1a6f95ed7b986ccb29fc69707788582fa672395abe3446e80",
   "prover_size": 75949172,
-  "verifier_checksum": "1291734485f70e932337080b7724a3ad49a965c5e2120ed3e47521c75cbf8c83",
-  "verifier_size": 665
+  "verifier_checksum": "86e2667e415b0e6266e0e9f3edb99746837debca70e9e82fd7d55f732ff7d995",
```

Comment:
Why do all of these irrelevant transition functions change parameters?

Reply:
Your intuition should give off a major red flag if you see behavior like this, and it should be documented if this change is well-founded.

Reply:
Good catch. As for a follow up from @raychu86:

```
 It’s not a big deal thankfully, especially since the proving keys are untouched. TLDR - it’s because of this PR -  https://github.com/AleoNet/snarkVM/pull/2444 that adds the 8 bytes of num_variables to the end the wrapper VKs. But this doesn’t seem to fail because we use the underlying Varuna VK when loading credits.aleo VKs from bytes.
 ```